drainer/checkpoint/checkpoint.go

```diff
@@ -19,14 +19,6 @@ import (
 	"go.uber.org/zap"
 )
 
-const (
-	// StatusConsistent means server quit normally, data <= ts is synced to downstream
-	StatusConsistent int = 0
```

Comment:
`Status` is more flexible and scalable. I think It's better if we map status to readable words.

Reply:
what about keep the origin Status and add a filed `StatusMessage` in the checkpoint ?